go/go.sum

```diff
@@ -14,6 +14,8 @@ github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd/go.mod h1:6dJ
 github.com/modern-go/reflect2 v1.0.1/go.mod h1:bx2lNnkwVCuqBIxFjflWJWanXIb3RllmbCylyMrvgv0=
 github.com/plaid/plaid-go v0.0.0-20190618232219-dafb04dae7bf h1:leSYdIy6gK+o/KAep9C9a93+qQzpCYgBoVyGKT6XMeA=
 github.com/plaid/plaid-go v0.0.0-20190618232219-dafb04dae7bf/go.mod h1:c7cDT1Lkcr0AgKJGVIG+oCa07jOrrg4Um8nduQ1eQN0=
+github.com/plaid/plaid-go v0.0.0-20200128010337-fe831fc21a4d h1:rhKSm00M5eHoFItsFbLaY0pIrTwLZeAxINDHx1uxKTg=
+github.com/plaid/plaid-go v0.0.0-20200128010337-fe831fc21a4d/go.mod h1:c7cDT1Lkcr0AgKJGVIG+oCa07jOrrg4Um8nduQ1eQN0=
```

Comment:
Running `go mod tidy` should remove the `v0.0.0-20190618232219-dafb04dae7bf` lines above.

Reply:
https://github.com/plaid/quickstart/pull/88/commits/5a990545dcbfb3f3e5e31346e876817b7abfd9a9